.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: github-project-issue-to-sheets
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  workflow_dispatch:
+  issues:
+    types: [opened, deleted, transferred, closed, reopened, assigned, unassigned, labeled, unlabeled]
+
+jobs:
+    github-project-issue-to-sheets:
+        runs-on: ubuntu-latest
+        name: github-project-issue-to-sheets
+        steps:
+        - name: github-project-issue-to-sheets
```

Comment:
suspicious hyphen here

Reply:
This is valid YAML/GH action syntax for a list as far as I can tell:

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idsteps
> A job contains a sequence of tasks called steps.